stylesheets/style.css

```diff
@@ -0,0 +1,2328 @@
+:root {
+  font-size: 1rem;
+}
+
+*,
+*::before,
+*::after {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+html {
+  margin: auto;
+  background-color: white;
+  box-sizing: border-box;
```

Comment:
You shouldn't have to declare the box-sizing if you already did that for all elements a few lines above.